examples/android/app/build.gradle

```diff
@@ -37,16 +37,18 @@ protobuf {
     generateProtoTasks {
         all().each { task ->
             task.builtins {
-                javanano {
-                    // Options added to --javanano_out
-                    option 'ignore_services=true'
+                // Javanano is installed by default, so we need to remove it like so
```

Comment:
Oh, misread, but yes, if we just have the `java { option 'lite' }`, it still builds as nano for some reason.